### ISSUE TITLE
What is the difference between user permissions for Admin and User interfaces?

### ISSUE THREAD

**Issue by daniel-z** [pencilblue/pencilblue#617]

**Comment by blakecallens** [comment]
Anything above USER (WRITER and above) has access to the admin section, but the admin section will be limited, based on the permission level.

WRITERs can only create articles.
EDITORs can create pages and do basic management, like create new writers
MANAGING_EDITORs are essentially admins, without code level access
ADMINISTRATORs are super users

**Comment by daniel-z** [comment]
Thanks for the answer, very clear. I have worked for a couple of weeks on pencilblue and I'm impressed how fast I could build themes, plugins and more on it.

**Comment by ElishaKay** [comment]
Hey Blake,
I discovered Pencilblue on Friday, and have been playing around with it for a day or two.
First of all - super awesome application.
One small issue:

I opened up a new account and had the admin give the new account "writer" status.
When I log in with my new account, I still can't access the articles feature.
Any idea what's causing this glitch?
Or, where to look for the solution.
Again, thanks so much!
